lib/internal/bridge/characteristics_mixin.dart

```diff
@@ -79,7 +79,7 @@ mixin CharacteristicsMixin on FlutterBLE {
         MethodName.writeCharacteristicForIdentifier,
         <String, dynamic>{
           ArgumentName.characteristicIdentifier: characteristicIdentifier,
-          ArgumentName.bytes: bytes,
+          ArgumentName.value: bytes,
```

Comment:
Why are there changes in FlutterBleLib?

Reply:
I created a separate PR for FlutterBleLib https://github.com/Polidea/FlutterBleLib/pull/232